src/pages/billing/List/ProjectListItem.ts

```diff
@@ -17,66 +17,95 @@ import { BillingProjectActions } from 'src/pages/billing/List/BillingProjectActi
 import { BillingProject } from 'src/pages/billing/models/BillingProject'
 
 
-const styles = {
-  projectListItem: selected => {
+const listItemStyle = (selected, hovered) => {
+  const style = {
+    ...Style.navList.itemContainer(selected),
+    ...Style.navList.item(selected),
+    ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
+    paddingLeft: '2rem'
+  }
+  if (hovered) {
     return {
-      ...Style.navList.itemContainer(selected),
-      ...Style.navList.item(selected),
-      ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
-      paddingLeft: '3rem'
+      ...style,
+      ...Style.navList.itemHover(selected)
     }
+  } else {
+    return style
   }
 }
 
-interface ProjectListItemProps {
+export interface ProjectListItemProps {
   project: BillingProject
   loadProjects: () => void
   isActive: boolean
-  isCreatingStatus: boolean
+  isCreating: boolean
+  isDeleting: boolean
 }
 
 export const ProjectListItem = (props: ProjectListItemProps) => {
+  const [hovered, setHovered] = useState<boolean>()
```

Comment:
Related to moving the actions outside of the link-- it's not good to have a button inside of a link.